### DISCUSSION TITLE
CTS Characterization

### DISCUSSION THREAD

**Discussion by EffeErreG4** [The-OpenROAD-Project/OpenROAD#1539] [Q&A] [answered]
I was trying to customize the CTS parameters using the command "configure_cts_characterization", but I don't see any changes is the maximum slew constrain reported in the .log file. Trying to understand if I was doing something wrong I noticed that there's a file called "CtsOptions.h" that should set the desired constrains with no "CtsOptions.cpp" file to describe the actual function. How can I manage to customize the maximum transition time? Thanks for your attention.

**Comment by maliberty** [comment, marked as answer]
All the functions are inlined so there is no need for a cpp.

Would you give the command you are using and the message you are getting in the log.

**Comment by EffeErreG4** [comment]
Sorry if I made a silly observation.
I was trying to define the maximum limits of slew and maybe I misunderstood the command.
Is it possible to change the values of maximum slew and maximum skew constrain for the CTS synthesis?
